sunpy_sphinx_theme/sunpy/static/style.css

```diff
@@ -433,6 +433,7 @@ div.figure p.caption span.caption-text {
 .card {
 	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
 	background-color: #F5F5F5;
+  margin: 0 auto;
```

Comment:
this makes the card float in the centre of the column, which is nice when the column takes up most of the screen.